src/config/constants/pools.ts

```diff
@@ -322,7 +322,7 @@ const pools: PoolConfig[] = [
     harvest: true,
     sortOrder: 999,
     tokenPerBlock: '0.009837',
-    isFinished: false,
+    isFinished: true,
```

Comment:
Can you confirm these are cool to show as `finished` @Chef-Chungus? We're past their endblock.